Firmware/LoRaSerial_Firmware/States.ino

```diff
@@ -563,6 +565,33 @@ void updateRadioState()
                    HEARTBEAT 1 ----> Update timestampOffset
     */
 
+    //====================
+    //Wait for the data transmission to complete
+    //====================
+    case RADIO_P2P_LINK_UP_WAIT_TX_DONE:
+      if (timeToHop == true) //If the channelTimer has expired, move to next frequency
+        hopChannel();
+
+      if (transactionComplete)
+      {
+        transactionComplete = false; //Reset ISR flag
+
+        //Determine the next packet size for SF6
+        if (ackTimer)
+        {
+          //Waiting for an ACK
+          triggerEvent(TRIGGER_LINK_WAIT_FOR_ACK);
+          sf6ExpectedSize = headerBytes + CHANNEL_TIMER_BYTES + trailerBytes;
```

Comment:
Not that SF6 is a huge priority right now but CHANNEL_TIMER_BYTES has now moved into headerBytes.